polire/interpolate/natural_neighbors/natural_neighbors.py

```diff
@@ -162,7 +158,7 @@ def _predict(self, X):
                 result[index] = self.y[idx]
 
             else:
-                #QHull object can't be pickled. Deepcopy doesn't work. 
+                #QHull object can't bgit ae pickled. Deepcopy doesn't work. 
```

Comment:
comment issue here. not a big deal.

Reply:
I was doing git add and you can see what happened there.